mqtt/mqtt-bridge/tests/integration.rs

```diff
@@ -126,6 +127,86 @@ async fn send_message_upstream_downstream() {
     local_client.shutdown().await;
 }
 
+#[ignore = "This test can be activated once RingBuffer is merged in."]
+#[tokio::test]
+async fn send_message_upstream_with_crash_is_lossless() {
```

Comment:
The goal for this test is to simulate a crash and then come back up and if rb is configured then the messages pick off where they were before crash. If there are ideas on how to improve this test, let me know!

Reply:
It would be great if you can write up the scenario as a test comment. Take a look how it's done for other tests in this file.

Reply:
Yes, will do